scripts/check_license_headers.py

```diff
@@ -34,7 +34,7 @@
     # Exclude CI files.
     r"^\.(github|circleci)/"
     # Exclude images.
-    r"|\.(?:png|gif|ttf|woff|otf|eot|woff2|ico|svg)$"
+    r"|\.(?:jpg|png|gif|ttf|woff|otf|eot|woff2|ico|svg)$"
```

Comment:
why do we add `jpg` to excluded file extensions here?

I mean this is probably the correct thing to do, but is it because some jpg that was intended to be committed is missing?

Reply:
It's not needed in this PR, however it's very small fix, used by me while working on E2E tests, can we please add it since it's really small change unrelated to logic and definitely an improvement?